relay/polkadot/src/lib.rs

```diff
@@ -1109,7 +1050,7 @@ pub enum ProxyType {
 	Governance = 2,
 	Staking = 3,
 	// Skip 4 as it is now removed (was SudoBalances)
-	IdentityJudgement = 5,
+	// Skip 5 as it was IdentityJudgement
```

Comment:
This would make proxies undecodable? Probably no issue since we dont need those anymore but CI should complain...